architecture/additional_concepts/service_catalog.adoc

```diff
@@ -0,0 +1,321 @@
+quality-of-services (QoS), each with a different cost associated with it.
+
+Service Instance::
+A _service instance_ is a provisioned instance of a service class. These are
+created by cluster users who wish to make a new, distinct instance of some type
+of managed service to make that available for use by one or more of their
+applications. When a new `Instance` resource is created, the service catalog
+controller connects to the appropriate broker server and instructs it to
+provision the service instance.
++
+.Example `Instance` Specification
+----
+apiVersion: servicecatalog.k8s.io/v1alpha1
+kind: Instance
+metadata:
+  name: my_db
+spec:
+  serviceClassName: smallDB
+----
+
+Credentials::
+_Credentials_ are information needed by an application to communicate with a
+service instance.
+
+Service Binding::
+A _service binding_ is a link between a service instance and an application.
+These are created by cluster users who wish for their applications to reference
+and use a service instance. Upon creation, the service catalog controller
+creates a Kubernetes secret containing connection details and credentials for
+the service instance. Such secrets can be mounted into pods as usual.
++
+.Example `Binding` Specification
+----
+apiVersion: servicecatalog.k8s.io/v1alpha1
+kind: Binding
+metadata:
+  name: myBinding
+spec:
+  secretName: mySecret
+  <pod_selector_labels>
+----
+
+[[service-catalog-provided-brokers]]
+== Provided Service Brokers
+
+{product-title} provides the following service brokers for use with the service
+catalog.
+
+[NOTE]
+====
+Because the service catalog is currently Technology Preview, the following
+service brokers are also currently Technology Preview.
+====
+
+[[service-catalog-template-service-broker]]
+=== Template Service Broker
+
+The _template service broker_ gives the service catalog visibility into the
+xref:../../dev_guide/templates.adoc#using-the-instantapp-templates[default Instant App and Quickstart templates] that have shipped with {product-title}
+since its initial release. By default, the template service broker shows the
+objects that are globally available from the *openshift* project. It can also be
+configured to watch any other project that a cluster administrator chooses.
+
+ifndef::openshift-online,openshift-dedicated[]
+See _Deploying the Template Service Broker_ (TODO link) for more details.
+endif::[]
+
+[[service-catalog-asb]]
+=== Ansible Service Broker
+
+The Ansible service broker is an implementation of the OSB API that manages
+applications defined by xref:service-catalog-apb[_Ansible playbook bundles (APBs)_]. APBs provide a new method for defining and distributing container
+applications in {product-title}, consisting of a bundle of Ansible playbooks
+built into a container image with an Ansible runtime. APBs leverage Ansible to
+create a standard path for transitioning from easy to complex deployments.
+
+The design of the Ansible service broker follows this basic workflow:
```

Comment:
ASB has some design docs in the repo that we may consider linking to: https://github.com/openshift/ansible-service-broker/blob/master/docs/design.md